### PR TITLE
Two more Raspicam fixes

### PR DESCRIPTION
**Pull request by 6by9**
RaspiYuv didn't support -md to set the sensor mode.

Raspividyuv would stall if you didn't set an output file or it failed to open as the output port had already been enabled so was waiting for buffers.

### CODE REVIEW

**Comment by popcornmix** [comment]
Looks fine to me. @JamesH65 ?

File: host_applications/linux/apps/raspicam/RaspiVidYUV.c [hunk 1]
```diff
@@ -1525,15 +1525,6 @@ int main(int argc, const char **argv)
          if (state.verbose)
```
[the remark this answers is not in the document]
Comment (6by9) [reply host_applications/linux/apps/raspicam/RaspiVidYUV.c]: (Now that you've posted those comments!)
Yes, OK can do that.

**Comment by 6by9** [comment]
Requested changes made. Anything else?

**Comment by JamesH65** [comment]
LGTM

**Pull request merged**